### PR TITLE
feat(Logo): Refactor logo to improve bundle size and loading times

### PR DESCRIPTION
**Pull request by pladaria**
* Logos are now lazy loaded. This should avoid sending unneeded skins and SVG data to apps
* Make logo images RSC friendly

WEB-1986

### before

image

image

:warning: The logo chunk is 62kB

### after

image

No `logo` chunk in the bundle

### CODE REVIEW

File: src/__acceptance_tests__/__ssr_pages__/main-navigation-bar-large.tsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ const NavigationBarTest = (): JSX.Element => (
             title,
             to: `/${title}`,
         }))}
+        logo={<span>LOGO</span>}
```
Comment (pladaria) [inline src/__acceptance_tests__/__ssr_pages__/main-navigation-bar-large.tsx:18]: SSR tests fail with Lazy components.
Ticket to review this: https://jira.tid.es/browse/WEB-1998

File: src/logo-blau.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
```
Comment (pladaria) [inline src/logo-blau.tsx:1]: each logo has its own file with a default export (this is required for lazy imports)

**Review by atabel — APPROVED** [review]

File: src/logo.tsx [hunk 3]
```diff
@@ -3,380 +3,105 @@ import * as React from 'react';
...
+            './logo-telefonica'
+        )
+);
 
-const TuLogoImage = ({size}: LogoImageProps) => {
-    const {isDarkMode} = useTheme();
-    const isInverse = useIsInverseVariant();
-    const {colors} = getTuSkin();
-    const color = isDarkMode ? colors.inverse : isInverse ? colors.inverse : colors.backgroundBrand;
+const BlauLogoImage = React.lazy(
```
Comment (marcoskolodny) [inline src/logo.tsx:58]: I wouldn't use "Image" as suffix of these components.

I'd use something like BlauLogoComponent or InternalBlauLogo

**Review by marcoskolodny — APPROVED** [review]

File: src/logo.tsx (hunk 3 above)
Comment (pladaria) [reply src/logo.tsx:58]: Why not? this is the component that returns the logo image

Comment (marcoskolodny) [reply src/logo.tsx:58]: I misunderstood the changes. It looks good to me

**Pull request merged**